skada/deep/tests/test_deep_scorer.py

```diff
@@ -0,0 +1,74 @@
+# Author: Yanis Lalou <yanis.lalou@polytechnique.edu>
+#
+# License: BSD 3-Clause
+
+import numpy as np
+import pytest
+import torch
+
+from skada.deep.base import (
+    BaseDALoss,
+    DomainAwareCriterion,
+    DomainAwareModule,
+    DomainAwareNet,
+    DomainBalancedDataLoader,
+)
+from skada.deep.modules import ToyModuleClassification2D
+from skada.metrics import (
+    DeepEmbeddedValidation,
+    PredictionEntropyScorer,
+    SoftNeighborhoodDensity,
+)
+
+
+class TestLoss(BaseDALoss):
```

Comment:
If we use it in different tests maybe we can create a '_TestLoss` class somewhere